test/redirects/redirects.csv

```diff
@@ -38,6 +37,6 @@
 /dora-report-2015,/research/2017-and-earlier/2015-state-of-devops-report.pdf,302
 /dora-report-2014,/research/2017-and-earlier/2014-state-of-devops-report.pdf,302
 /next24,https://google.qualtrics.com/jfe/form/SV_8uCHA4aRzcGDjg2?source=next24,302
-/devops-capabilities/cultural/how-to-empower-software-delivery-teams/,/guides/how-to-empower-software-delivery-teams/,302
-/devops-capabilities/cultural/devops-culture-transform/,/guides/devops-culture-transform/,301
+/capabilities/how-to-empower-software-delivery-teams/,/guides/how-to-empower-software-delivery-teams/,302
+/capabilities/devops-culture-transform/,/guides/devops-culture-transform/,301
```

Comment:
`/devops-capabilities/cultural/devops-culture-transform/,/guides/devops-culture-transform/,301`
AND
`/capabilities/devops-culture-transform/,/guides/devops-culture-transform/,301`